easybuild/easyconfigs/a/ABINIT/ABINIT-7.10.5-intel-2016.02-GCC-4.9_libxc.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'ConfigureMake'
+
+name = 'ABINIT'
+version = "7.10.5"
+versionsuffix = "_libxc"
```

Comment:
we usually use a `-` in versionsuffixes, so use `-libxc` (and rename the easyconfig accordingly)?